app/controllers/api/v1/rooms_controller.rb

```diff
@@ -26,6 +26,31 @@ def show
         }, status: :ok
       end
 
+      # POST /api/v1/room/:friendly_id/start.json
+      # Returns: { data: Array[serializable objects] , errors: Array[String] }
+      # Does: Starts the Room session and joins in the room starter.
+
+      def start
+        # TODO: amir - Check the legitimately of the action.
+        bbb_api = BigBlueButtonApi.new
+        room_id = @room.friendly_id
```

Comment:
No need for this - I would just keep referring to it as `@room.friendly_id`

Reply:
Makes sense, will do it

Reply:
DONE